src/style.css

```diff
@@ -0,0 +1,99 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+.page {
+  --big-bar-height: 70px;
+  --big-bar-pleft: 62px;
+  --big-bar-border: 1px;
+  --big-search-icon-top: calc(var(--big-bar-height) / 2);
+
+  --small-bar-height: 42px;
+  --small-bar-pleft: 33px;
+  --small-bar-border: 1px;
+  --small-search-icon-top: calc(var(--small-bar-height) / 2);
+
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 300;
+}
+/*
+.page-body {
+  margin: 0;
+  padding: 0;
+} */
+
+.container {
+  margin-top: 20px;
+  position: relative;
+  display: block;
+  height: var(--big-bar-height);
+}
+
+.container--small {
+  height: var(--small-bar-height);
+}
+
+.search-icon {
+  position: absolute;
+  top: var(--big-search-icon-top);
+  left: 26px;
+  transform: translateY(-50%);
+  width: 19px;
+  height: 19px;
+}
+
+.search-icon--small {
+  top: var(--small-search-icon-top);
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+.bar {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: #fff;
+  border: var(--big-bar-border) solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-size: 16px;
+  width: 100%;
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 300;
+
+  display: block;
+  padding-left: var(--big-bar-pleft);
+  height: var(--big-bar-height);
```

Comment:
instead of setting the height explicitly, try to set the correct padding, font size, and line-height. the height will then be calculated automatically

Reply:
I have to set the height because the clearance from the top for my icons depends on it. When I set the height by padding and font size I to hardcode the position of my absolutely placed icons.

Reply:
if the icons are positioned using absolute, don't they already have hardcoded values for top and left?

Reply:
Okay, I fixed.